.gitignore

```diff
@@ -36,3 +36,5 @@ test/nightwatch_tests/selenium/*
 *.react.js
 selenium-debug.log
 npm-debug.log
+test/bundle.js
+test/templates.js
```

Comment:
This is a tricky one. If we move them to dist then we cannot have the dev server running and run the tests at the same time.